test/module.test.js

```diff
@@ -2,6 +2,7 @@ import { resolve } from 'path'
 import { readFileSync } from 'fs'
 import { generate, setup, loadConfig, get, url } from '@nuxtjs/module-test-utils'
 import { JSDOM } from 'jsdom'
+import { withoutTrailingSlash, withTrailingSlash } from 'ufo'
```

Comment:
And this as well?

Reply:
This is OK in the tests and in the context of the module since then the normal node importing mechanism will ensure that the project-specific version of ufo is picked up.

It's only a problem for webpack context - the files that are added through `addTemplate` or `addPlugin`.

Reply:
@rchl thank you very much for making it clear!